src/main/java/org/sefglobal/scholarx/service/SimpleJavaMail.java

```diff
@@ -0,0 +1,27 @@
+package org.sefglobal.scholarx.service;
+
+import org.sefglobal.scholarx.model.Mail;
+import org.simplejavamail.api.email.Email;
+import org.simplejavamail.api.mailer.Mailer;
+import org.simplejavamail.email.EmailBuilder;
+
+public class SimpleJavaMail {
+
+    private final MailInstance mailInstance;
+
+    public SimpleJavaMail(MailInstance mailInstance) {
+        this.mailInstance = mailInstance;
+    }
+
+    public void sendEmail(Mail mail) {
+
+        Email email = EmailBuilder.startingBlank()
+                .to(mail.getName(), mail.getEmailAddress())
+                .withSubject(mail.getSubject())
+                .withHTMLText(mail.getMessage())
+                .buildEmail();
+
+        Mailer mailer = mailInstance.getInstance();
+        mailer.sendMail(email);
+    }
+}
```

Comment:
We no longer need this, right?

Reply:
It needed. I moved the singleton connection (Mail instance) to another class so I kept this file as it is.

Reply:
No. Let's move this logic to the EmailUtil Class.